python/python/glide/glide_client.py

```diff
@@ -516,6 +518,31 @@ class GlideClusterClient(BaseClient, ClusterCommands):
     https://github.com/aws/babushka/wiki/Python-wrapper#redis-cluster
     """
 
+    async def _cluster_scan(
+        self,
+        cursor: ClusterScanCursor,
+        match: Optional[str] = None,
+        count: Optional[int] = None,
+        type: Optional[ObjectType] = None,
```

Comment:
Doc: `TYPE option to ask SCAN to only return objects that match a given type`
The parameter is saying what is the type of the object you want to get.
Also, i would like to stay consistent with the impl in redis-rs.

Reply:
The server is scanning and checking if the object match a given *data* type. The type that the user passes is the data type that all objects should match with.. 
also doc:
> Why SCAN may return all the items of an aggregate data type in a single call?
In the COUNT option documentation, we state that sometimes this family of commands may return all the elements of a Set, Hash or Sorted Set at once in a single call, regardless of the COUNT option value. The reason why this happens is that the cursor-based iterator can be implemented, and is useful, only when the aggregate data type that we are scanning is represented as a hash table. However Redis uses a [memory optimization](https://redis.io/docs/latest/operate/oss_and_stack/management/optimization/memory-optimization/) where small aggregate data types...

+ https://redis.io/docs/latest/develop/data-types/

so IMO in the context of redis 'data type' is the more accurate way to name it, and I also think that in redis-rs you should have called it DataType..
However if it's a large change since it is already merged in redis-rs it's up to you if you want to keep it or change it